internal/pkg/template/templates/environment/cf.yml

```
@@ -189,6 +189,24 @@ Resources:
       Tags:
         - Key: Name
           Value: !Sub 'copilot-${AppName}-${EnvironmentName}-env'
+{{- if and .VPCConfig.SecurityGroupConfig (gt (len .VPCConfig.SecurityGroupConfig.Ingress) 0)}}
+      SecurityGroupIngress:
+      {{range $securityRule := .VPCConfig.SecurityGroupConfig.Ingress}}
```

To remove extra new lines (same for egress)
```suggestion
      {{- range $securityRule := .VPCConfig.SecurityGroupConfig.Ingress}}
```